torch/testing/_internal/common_methods_invocations.py

```diff
@@ -171,10 +191,20 @@ def get_inplace(self):
         """
         return self.inplace_variant
 
+    def get_operator_variant(self):
+        """Returns operator variant of the operator, e.g. operator.neg
+        Returns None if the operator has no operator variant.
+        """
+        return self.operator_variant
+
     def sample_inputs(self, device, dtype, requires_grad=False):
         """Returns an iterable of SampleInputs."""
         return self.sample_inputs_func(self, device, dtype, requires_grad)
 
+    def sample_sparse_inputs(self, device, dtype, requires_grad=False, **kwargs):
+        """Returns an iterable of SampleInputs."""
```

Comment:
This docstring should clarify the intent of these SampleInputs, as opposed to sample_inputs() above, is for use in testing the sparse version of these operations.